socialhome/streams/app/components/StreamElement.vue

```diff
@@ -25,7 +25,9 @@
                 </div>
             </div>
             <div class="ml-auto grid-item-reactions mt-1">
-                <div v-if="showShares" class="item-reaction" @click.stop.prevent="expandShares">
+                <div v-if="showShares" class="item-reaction" v-bind:class="{'item-reaction-shared': getHasShared}"
```

Comment:
Just do `:class="{'item-reaction-shared': getHasShared}"`. `:` is a shorthand to `v-bind`.

Reply:
Ah, nice! 👍